### PR TITLE
error in gallery + images getting copied to next pdf

### PR DESCRIPTION
**Pull request by ayyysh04**
fixed the bugs, please check the same @anushbhatia

### CODE REVIEW

**Comment by anushbhatia** [comment]
Can you change the heading to white color that would be great. Rest amazing work 🥳

**Comment by ayyysh04** [comment]
> 
> 
> Can you change the heading to white color that would be great. Rest amazing work 🥳

I have not changed the theme constant, In my phone its showing appbar text in white only. You can check here
`class AppTheme {
  static const Color primaryColor = Colors.white;
}`

**Comment by ayyysh04** [comment]
I have not changed the theme constant, In my phone its showing appbar text in white only. You can check here
`
class AppTheme 
{ static const Color primaryColor = Colors.white;
 }`

File: lib/constants/theme_constants.dart [hunk 1]
```diff
@@ -47,15 +56,21 @@ final lightTheme = ThemeData(
...
-          fontSize: 22.0,
-          fontWeight: FontWeight.w700,
-        ))));
+      elevation: 0,
+      color: Colors.blue[600],
+      iconTheme: const IconThemeData(
+        color: AppTheme.primaryColor,
+      ),
+      titleTextStyle: const TextStyle(
+        color: AppTheme.primaryColor,
```
Comment (anushbhatia) [inline lib/constants/theme_constants.dart]: Check this once i think this should be white.

**Review by anushbhatia — CHANGES_REQUESTED** [review]

**Comment by ayyysh04** [comment]
Done, please check the same @anushbhatia

**Comment by anushbhatia** [comment]
Awesome Work @ayyysh04 looking forward to more contributions from your end.
Alt Text

**Pull request merged**